### PR TITLE
Add some codeowners

### PR DESCRIPTION
**Pull request by radical**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/4615)

### CODE REVIEW

**Comment by radical** [comment]
I added a few, but please free to add more.

**Review by eerhardt — APPROVED** [review]
LGTM

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1,16 @@
-/eng/SourceBuild* @dotnet/source-build-internal
\ No newline at end of file
+/eng/SourceBuild* @dotnet/source-build-internal
```
Comment (radical) [inline CODEOWNERS:1]: Why is github complaining about this? This line is same as in the other repos - https://github.com/dotnet/sdk/blob/7822425c3ea35e0bb46e57fad11663e7cbf9f850/CODEOWNERS#L106

Comment (radical) [reply CODEOWNERS:1]:
```
Unknown owner on line 1: make sure the team @dotnet/source-build-internal exists, is publicly visible, and has write access to the repository
/eng/SourceBuild* @dotnet/source-build-internal
```

Comment (DamianEdwards) [reply CODEOWNERS:1]: Has that team been given any rights to this repo? I don't see it in the list.

Comment (radical) [reply CODEOWNERS:1]: I guess not! Who can fix this?

Comment (radical) [reply CODEOWNERS:1]: cc @joperezr

**Pull request merged**